ui/src/pages/OBProxy/Detail/Overview/ConfigDrawer.tsx

```diff
@@ -0,0 +1,206 @@
+        </CustomFormItem>
+        <CustomFormItem name="replicas" label="副本数">
+          <InputNumber placeholder="请输入" min={1} />
+        </CustomFormItem>
+        <CustomFormItem name={['resource', 'cpu']} label="CPU 核数">
+          <InputNumber placeholder="请输入" min={1} />
+        </CustomFormItem>
+        <CustomFormItem name={['resource', 'memory']} label="内存大小">
+          <InputNumber placeholder="请输入" min={1} addonAfter={SUFFIX_UNIT} />
+        </CustomFormItem>
+        <p style={titleStyle}>参数设置</p>
+        <Form.List name={'parameters'}>
+          {(fields, { add, remove }) => (
+            <>
+              {fields.map(({ name }) => (
+                <Row key={name} gutter={[12, 0]}>
+                  <Col span={12}>
+                    <CustomFormItem name={[name, 'key']}>
+                      <Select
+                        placeholder="请选择"
+                        onChange={(val) => labelChange(val, name)}
+                        optionLabelProp="selectLabel"
+                        options={listParametersOptions?.map((item) => ({
+                          value: item.value,
+                          selectLabel: item.label,
+                          label: (
+                            <div
+                              style={{
+                                display: 'flex',
+                                justifyContent: 'space-between',
+                              }}
+                            >
+                              <Text>{item.label}</Text>
+                              <Text
+                                ellipsis={{ tooltip: item.info }}
+                                style={{ color: '#8592AD', width: 120 }}
+                              >
+                                {item.info}
+                              </Text>
+                            </div>
+                          ),
+                        }))}
+                      />
+                    </CustomFormItem>
+                  </Col>
+                  <Col span={11}>
+                    <CustomFormItem name={[name, 'value']}>
+                      <Input />
+                    </CustomFormItem>
+                  </Col>
+                  <Col
+                    style={{
+                      lineHeight: '32px',
+                      textAlign: 'center',
+                    }}
+                    span={1}
+                  >
+                    <MinusCircleOutlined onClick={() => remove(name)} />
+                  </Col>
+                </Row>
+              ))}
+              <CustomFormItem>
+                <Button
+                  type="dashed"
+                  onClick={() => add()}
+                  block
+                  icon={<PlusOutlined />}
+                >
+                  {intl.formatMessage({
+                    id: 'OBDashboard.components.NodeSelector.AddNodeselector',
+                    defaultMessage: '添加 nodeSelector',
```

Comment:
By now there is no `nodeSelector` configuration item in backend. Had better remove this item.